src/cli.js

```diff
@@ -169,6 +206,14 @@ function parseArgs(argv) {
       describe:
         "Local path or URL of schema to validate file against. If not supplied, we will attempt to find an appropriate schema on schemastore.org using the filename",
     })
+    .option("catalogs", {
+      type: "string",
+      alias: "c",
+      array: true,
+      describe:
+        "Local paths of custom catalogs to use prior to schemastore.org",
```

Comment:
This can also be a URL

Reply:
Fixed in 7b3428af9bfd65407dfceebcb7e6bd12de6de600